src/reanimated2/animations.js

```diff
@@ -21,7 +21,7 @@ export function transform(value, handler) {
   if (typeof value === 'string') {
     // toInt
     // TODO handle color
-    const match = value.match(/([A-Za-z]*)([-\d.]*)([A-Za-z]*)/);
+    const match = value.match(/([A-Za-z]*)([-\d.]*)([A-Za-z%]*)/);
```

Comment:
This regex is wrong, `([-\d.]*)` allows parsing `-1-...1` as a number.
I think `(-?\d*\.?\d*)` is slightly better, it matches the optional minus sign, and optional dot between numbers (see attached screenshot).
![image](https://user-images.githubusercontent.com/12465392/104195503-fe7a9600-5422-11eb-82e2-8fa06a3f141f.png)